packages/mantine/src/components/RadioCard/RadioCard.module.css

```diff
@@ -0,0 +1,62 @@
+.card {
+    display: inline-flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: var(--mantine-spacing-md);
+    border-radius: var(--mantine-radius-md);
+    box-shadow: var(--mantine-shadow-md);
+    transition: all 150ms ease;
+
+    &:hover:not(:disabled) {
+        background-color: var(--mantine-color-default-hover);
+        box-shadow: var(--mantine-shadow-xl);
+    }
+
+    &[data-checked]:not(:disabled) {
+        border-color: var(--mantine-primary-color-filled);
+        box-shadow: var(--mantine-shadow-xs);
+
+        &:hover {
+            box-shadow: var(--mantine-shadow-xl);
+        }
+    }
+
+    &:disabled {
+        box-shadow: var(--mantine-shadow-xs);
+        cursor: not-allowed;
+
+        .title {
+            color: var(--mantine-color-disabled-color);
+        }
+
+        .description {
+            color: var(--mantine-color-disabled-color);
+        }
+    }
+}
```

Comment:
I know you probably just copy pasted what we had, but seing the current implementation it would make more sense to reuse the Card component styles somehow.

<img width="295" height="427" alt="image" src="https://github.com/user-attachments/assets/ac442a54-3b46-44f3-bdc4-e6ac979abc22" />

vs

<img width="615" height="398" alt="image" src="https://github.com/user-attachments/assets/c783734c-f440-4cf5-9f49-fdacf44027a0" />


The shadow is much more pronounced in the RadioCard

Perhaps if we set prop `component={Card}` on the `Radio.Card` component it would work.